exercises/APE1/0103

```diff
@@ -0,0 +1,114 @@
+Hence, pirate $p_{5}$ will propose 0 to $p_{1}$, 2 to $p_{2}$, 1 for $p_{3}$, 0 for $p_{4}$ and 7 for himself. Pirates $p_{4}$ and $p_{1}$ will not vote for that proposition, but pirates $p_{2}$ and $p_{3}$ will clearly vote for that proposition, hence we have found the repartition.
+
+\paragraph{Putting everything together.} The final decisions regarding the division of the gold are given on Table \ref{tab:pirates5}. On this table, each horizontal block shows the division of the gold, in function of the value of $n$, where $n$ is the number of pirates in the crew.
+
+
+\begin{tabular}[h!]{|lllll||r|}
+\hline
+&         &           &                    &                     & $n$ \\ \hline
+&         &           &                    & $p_{1}$             & 1   \\
+&         &           &                    & 10                  &     \\ \hline
+&         &           & $p_{2}$            & $p_{1}$             & 2   \\
+&         &           & \textcolor{red}{0} & \textcolor{red}{10} &     \\ \hline
+&         & $p_{3}$   & $p_{2}$            & $p_{1}$             & 3   \\
+&         & 10        &  0                 & 0                   &     \\ \hline
+& $p_{4}$ & $p_{3}$ & $p_{2}$              & $p_{1}$             & 4   \\
+& 8       &  0        & 1                  & 1                   &     \\ \hline
+$p_{5}$ & $p_{4}$ & $p_{3}$   & $p_{2}$            & $p_{1}$             & 5   \\
+7       &  0      & 1         & 2                  &  0                  &     \\ \hline
+\end{tabular}
+\captionof{table}{Gold divisions for $n$ pirates, with $1 \leq n \leq 5$}
+\label{tab:pirates5}
+
+
+\vspace{5mm}
+
+
+Now we can think about what happens if we consider even more pirates. Let us complete the above table for values up to $n = 7$. We find the results given in Table \ref{tab:pirates7}.
+
+
+
+\begin{tabular}[h!]{|lllllll||r|}
+\hline
+&         &         &         &         &         &         & $n$ \\ \hline
+&         & $p_{5}$ & $p_{4}$ & $p_{3}$ & $p_{2}$ & $p_{1}$ & 5   \\
+&         & 7       & 0       & 1       & 2       & 0       &     \\ \hline
+& $p_{6}$ & $p_{5}$ & $p_{4}$ & $p_{3}$ & $p_{2}$ & $p_{1}$ & 6   \\
+& 6       & 0       & 1       & 2       &  0      & 1       &     \\ \hline
+$p_{7}$ & $p_{6}$ & $p_{5}$ & $p_{4}$ & $p_{3}$ & $p_{2}$ & $p_{1}$ & 7   \\
+6       &  0      & 1       & 2       & 0       & 1       & 0       &     \\
+\hline
+\end{tabular}
+\captionof{table}{Gold divisions for $n$ pirates, with $5 \leq n \leq 7$}
+\label{tab:pirates7}
+
+
+\vspace{5mm}
+
+
+Now we can think about what happens if we consider $n$ pirates. Let us give some notations: if the crew has $n$ pirates, then we know that pirate $p_{n}$ will propose a division of the gold. In this proposition, we define $a_{i}^{n}$ as the number of gold coins that pirate $i$ receives, for $i = 1, \dots, n$. The function that pirate $p_{n}$ wants to maximize is
+
+\begin{equation*}
+\sum_{i = 1}^{n} 2^{i} a_{i}^{n}
```

Comment:
I would be good to explain where this objective comes from. If I understand correctly, you are doing this to take into account the fact that the captain prefers giving coins to the highest ranks. However since there is only a factor 2 between consecutive ranks it is not obvious to me that it gives what we want. With 11^{i} instead of 2^{i}, and a small explanation that should be convincing enough :)